### DISCUSSION TITLE
iPadOS 16.2 - 2nd screen broken on stage manager

### DISCUSSION THREAD

**Discussion by TobiaszCudnik** [blinksh/blink#1652] [Support] [answered]
### Checklist

- [x] Using latest version on the [App Store](http://itunes.apple.com/app/id1156707581)
- [x] Read the [docs](http://blink.sh/docs/)
- [x] Searched for [existing GitHub issues](https://github.com/blinksh/blink/issues)

### Configuration

iPadOS 16.2 beta 4

### Describe the bug

1. Enable stage manager
1. Enable external display (extended, not mirror)

**Result**: Blink renders on both screens and the cursor cant be moved to the secondary screen.

**Expected**: One instance of Blink renders only on one screen and the cursor isnt affected.

**Comment by yury** [comment, marked as answer]
You should disable external display in blink (set to mirror) to be able to use blink in Stage Manager mode. 
It is a little bit confusing, we may detect Stage Manager mode.

**Comment by TobiaszCudnik** [comment]
I can confirm that `config -> Appearance -> Layout -> External Display = Mirror` and a restart indeed get the job done. Thanks!
